src/index.html

```diff
@@ -11,9 +11,42 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
 </head>
 <body>
-
+  <header class="main-header">
+    <h1>
```

Comment:
This is not an `h1`. Usually, the logo is a link to the homepage. So better to write:
```html
<a href="#" title="Go Home">
  <img ... >
</a>
```